### ISSUE TITLE
Game failing to launch on Mojave 10.14.6

### ISSUE THREAD

**Issue by yamyamyamyamyam** [ValveSoftware/Dota-2#1936]
#### Your system information

* System information from steam: https://gist.github.com/yamyamyamyamyam/6bb821f88e102051920439822b7d183f
* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: Yes. Also reinstalled the game completely and re-verified.

#### Please describe your issue in as much detail as possible:

The game is failing to start up after the most recent client update (I believe May 25). It reaches the point where the red Dota2 logo is visible, then immediately exits. There are no error codes or "unexpectedly quit" messages and the system isn't popping up with a crash report, so it's unclear what exactly is causing the game to quit.

**Comment by HadHod** [comment]
Same issue described here https://github.com/ValveSoftware/Dota-2/issues/1934

**Comment by yamyamyamyamyam** [comment]
Yes, it appears to be similar, but I created a separate issue since my machine isn't hanging or getting stuck, but rather quitting (and the different OS).

Here is a log of all messages in my console while the game attempts to launch (taken from the moment I launch it in Steam until the game finishes quitting). https://gist.github.com/yamyamyamyamyam/8f7ab924e96470de3be97ebc0a4675ca

If there's anything else I can provide to help please let me know!

**Comment by yamyamyamyamyam** [comment]
A workaround that allows Dota to launch on my system (I found this on the Dota2 subreddit) is launching dota.sh from the game files with the following command:

`~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/dota.sh`

I am not sure if that same launch process exists on Linux but if someone is trying to launch the game on Linux they might look into this workaround as well.

**Comment by Turmfalke2** [comment]
It does exist and I think if you launch it like that you launch it without the steam runtime environment. This might indicate that it is an issue with the runtime environment.
